docs: add platform docs (#7)
seqeralabs/nf-training-intro: docs/part4-platform.md

# Part 4. Critter classification on Seqera Platform

## Introduction to Running a Nextflow Workflow on Seqera Platform

Transitioning from crafting our classification steps into Nextflow and running locally, we're now stepping up to Seqera Platform, making our critter-classification project even more collaborative and accessible. This move isn't just about scaling up; it's about bringing collaborators into our fun classification journey, enabling anyone, anywhere, to classify their favorite critters using the same parameters. It's also about effortlessly handling data in the Cloud, without getting tangled in the technicalities, and keeping an eye on costs, ensuring our animal adventure stays budget-friendly. Nextflow Tower is here to make sharing, collaborating, and cloud computing a breeze for our critter-classifying escapades.

## Running the workflow on Seqera Platform

## Step 1: Log in to Seqera Platform

> Head over to our beloved, [Seqera Platform landing page](cloud.seqera.io) and log in with your Seqera GSuite account. Upon logging in, you will have access to Workspace called `nf-training` under the `seqeralabs` organization.

Reviewer: ```suggestion
Head over to our beloved [Seqera Platform landing page](cloud.seqera.io) and log in with your Seqera GSuite account. Upon logging in, you will have access to Workspace called `nf-training` under the `seqeralabs` organization.
```

> An AWS Batch Compute Environment with valid credentials has been already setup for you to use.

Reviewer: ```suggestion
An AWS Batch Compute Environment with valid credentials has been already set up for you to use.
```

As you are brought to the 'Pipeline Parameters' page, you will have the option of specifying which images you can use for classification, based on the `params.input` parameter. In the previous steps, we were using the few images that were provided for you. Let's make this more interesting...

## Step 3: Use data on cloud storage as input

In a valiant effort akin to treasure hunting through the digital realms, we've meticulously combed through the delightful array of critter images found in Seqera's #social-pet-pics channel. These gems have now found a new home in an AWS S3 bucket, eagerly awaiting their moment of fame to be classified into 'cat' or 'dog' versus 'cute cat' and 'cute dog'.

Your goal will be to classify the images in this bucket, in the Cloud.

In the 'input' field on the Parameters page, you can specify the following path to the bucket:

```
s3://seqera-development-permanent-bucket/scidev/petpics
```

## Step 4: Specify your prompts for classification

Similar to what you had done before, specify a list of labels you want to use to classify the images. For example, 'dog,cat,cute dog,beautiful dog'.

> ## Step 5: Specify where you results will be saved

Reviewer 2: ```suggestion
## Step 5: Specify where your results will be saved
```
